src/main/java/net/minecraftforge/fml/unsafe/UnsafeHacks.java

```diff
@@ -97,5 +97,6 @@ public static void cleanEnumCache(Class<? extends Enum<?>> enumClass) throws Exc
     {
         findField(Class.class, "enumConstantDirectory").ifPresent(f -> setField(f, enumClass, null));
         findField(Class.class, "enumConstants").ifPresent(f -> setField(f, enumClass, null));
+        findField(Class.class, "enumVars").ifPresent(f -> setField(f, enumClass, null));//Open J9
```

Comment:
the other lines of code here dont have comments. this one doesnt really need one either.

Reply:
The other lines of code assume a standard JDK, this one does not.